sql/telemetry_derived/main_summary_v4/part1.sql

```diff
@@ -180,16 +180,14 @@ SELECT
   -- buckets for these particular histograms is not fine enough for the median
   -- to give a more accurate value than the mean.
   udf.histogram_to_mean(
-    udf.json_extract_histogram(payload.histograms.places_bookmarks_count)
+    mozfun.hist.extract(payload.histograms.places_bookmarks_count)
```

Comment:
I believe the biggest single contributor to "resources exceeded" is the number of persistent UDF references. This PR removes all references to `udf.json_extract_histogram` with references to the underlying mozfun.hist.extract, eliminating a level of indirection.